playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -1,33 +1,49 @@
-import { test, expect } from '@playwright/test';
+import { test, expect} from '@playwright/test';
+import KsmodalPage from './Ksmodal.page';
 
-test('open rex page', async ({ page }) => {
-    await page.goto('https://staging.openstax.org/books/physics/pages/preface');
-    expect(page.url()).toBe('https://staging.openstax.org/books/physics/pages/preface');
-  });
+test('open keyboard shortcut modal using keyboard @testrail id: C651124', async ({ page }) => {
+  
+  // GIVEN: Open Rex page and tab 3 times 
+  await page.goto('/books/business-ethics/pages/preface');
+  await page.locator('body').press('Tab');
+  await page.locator('text=Skip to Content').press('Tab');
+  await page.locator('text=Go to accessibility page').press('Tab');
```

Comment:
Shouldn't this be `await page.keyboard.press('Tab')` for keypress 2 and 3 to show page navigation by keyboard? As written, it looks for each element, presses the tab key, and then ignores the new location. Same issue with line 13 below.